flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/SplitStream.scala

```diff
@@ -28,6 +28,7 @@ import org.apache.flink.streaming.api.datastream.{ SplitStream => SplitJavaStrea
  * To apply a transformation on the whole output simply call
  * the appropriate method on this stream.
  */
+@Deprecated
```

Comment:
Scala's deprecated annotation is `@deprecated`.

Moreover, could we add a deprecation message explaining why it shouldn't be used anymore? `@deprecated("reason", "deprecated since.")`